src/scripts.js

```diff
@@ -255,7 +284,7 @@ function displaySelectedRecipe(e) {
       return currentRecipe.id === recipeID;
     });
 
-    show([selectedRecipeView, homeButton, favoritePageButton]);
+    show([selectedRecipeView, homeButton, favoritePageButton, cookbookPageButton]);
```

Comment:
We'd forgotten earlier that we want to see the cookbook button on this view as well.